source/examples/rapids-azureml-hpo/notebook.ipynb

```diff
@@ -282,70 +217,58 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Create Experiment\n",
-    "\n",
-    "Track all the runs in your workspace"
+    "## Setup Environment"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 6,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "experiment_name = \"test_rapids_aml_cluster\""
+    "We'll be using a custom RAPIDS docker image to [setup the environment](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-manage-environments-v2?tabs=python#create-an-environment-from-a-docker-image). This is available in `rapidsai/base` repo on [DockerHub](https://hub.docker.com/r/rapidsai/base/)."
    ]
   },
   {
-   "cell_type": "markdown",
-   "metadata": {},
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
    "source": [
-    "## Setup Environment"
+    "%%bash\n",
+    "# create a Dockerfile defining the image the code will run in\n",
+    "cat > ./Dockerfile <<EOF\n",
+    "FROM {{ rapids_container }}\n",
+    "\n",
+    "RUN conda install --yes -c conda-forge 'dask-ml>=2024.4.4' \\\n",
+    " && pip install azureml-mlflow\n",
+    "EOF"
```

Comment:
This Dockerfile is so minimal... proposing here that we just inline the creation of it in the notebook, to remove the friction of having to create a file and populate it.

This also has the nice side benefit of putting it through the templating we do when rendering the docs, so the base image will now get updates (notice the in the deleted file `rapids-azureml-hpo/Dockerfile`, it's using RAPIDS 23.02 😱 ).